gravitee-am-cypress/cypress/integration/oidc/ciba/ciba.spec.ts

```diff
@@ -0,0 +1,175 @@
+                                    .then(() => {
+                                        startDomain(domain, accessToken)
+                                            .should(response => expect(response.status).to.eq(200))
+                                            .then(() => {
+                                                createCibaApplication(domainHrid)
+                                                    .should(response => expect(response.status).to.eq(201))
+                                                    .then(response => {
+                                                        clientId = response.body.client_id;
+                                                        clientSecret = response.body.client_secret;
+                                                        registerDomainWithNotifier(domain, domainHrid, clientId, clientSecret)
+                                                            .should(response => expect(response.status).to.eq(200));
+                                                    });
+                                            });
+                                    });
+                            });
+                        createUser(domain, accessToken, user).should(response => expect(response.status).to.eq(201));
+                    })
+            })
+    });
+
+    after(() => {
+        requestDomain(domain, accessToken, "DELETE", null)
+            .should(response => expect(response.status).to.eq(204)
+            )
+    })
+
+    context("The application requests authentication", () => {
+        it("AM responds with an auth_req_id", () => {
+            let loginHint = user.username;
+            let userCode = user.additionalInformation["user_code"];
+            requestAuthentication(domainHrid, clientId, clientSecret, loginHint, userCode).should(response => {
+                expect(response.status).to.eq(200);
+                expect(response.body.auth_req_id).to.not.be.null;
+                expect(response.body.expires_in).to.not.be.null;
+                expect(response.body.interval).to.eq(cibaConfig.oidc.cibaSettings.tokenReqInterval);
+            }).then(response => {
+                auth_req_id = response.body.auth_req_id
+            });
+        })
+    });
+
+    context("The application requests an access token when the user has accepted", () => {
+        it('should receive an access token', () => {
+            cy.request({
+                url: Cypress.env().gatewayUrl + `/${domainHrid}/oauth/token`,
+                method: 'POST',
+                auth: {
+                    user: clientId,
+                    password: clientSecret
+                },
+                form: true,
+                body: {
+                    auth_req_id: auth_req_id,
+                    grant_type: "urn:openid:params:grant-type:ciba",
+                }
+            }).should(response => {
+                expect(response.status).to.eq(200);
+            });
```

Comment:
Since the test is called 'should receive an access token' you probably want to verify that it actually receives a token (`response.body.access_token`), in addition to status code 200.